In [1]:
import pandas as pd 
import numpy as np

In [32]:
df_merge = pd.read_csv('./merge_data/train/train.csv' )
df_og = pd.read_csv('./data/train.csv') 

print(len(df_merge))
print(len(df_og))

In [44]:

df_merge_list = df_merge.image_id.values
df_og_list = df_og.image_id.values

item_list = []
for item in df_og_list:
    if item not in df_merge_list:
        item_list.append(item)

print(len(item_list))

375


## Current tasks

- Figure out why 375 is not contains into the original dataset, find them, make a new folder and check them what knid of pics of them.

- If there is no problem, create a new dataset which is contains the whole data we current have.

- Make EDAs of the orginal dataset and New dataset which we make, comapre them.

- Find more notebook about that (Make sure) Is there any notebook talking about the dataset?